client/tests/sdk/test_api.py

```diff
@@ -243,7 +244,7 @@ def read(self, n=-1):
                             "content-type": "application/pgp-encrypted",
                             "etag": "sha256:3aa5ec3fe60b235a76bfc2c3a5c5d525687f04c1670060632a21b6a77c131f65",  # noqa: E501
```

Comment:
Should we also be recalculating this? I guess the hash checking is done at a later stage that doesn't apply here.

Reply:
Exactly: that's a higher-level check than this test.